net.shrew.ike.qikea.json

```diff
@@ -0,0 +1,101 @@
+{
+    "app-id": "net.shrew.ike.qikea",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "command": "qikea",
+    "sdk": "org.freedesktop.Sdk",
+    "finish-args": [
+        "--filesystem=host",
+        "--filesystem=/run/ikedi",
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11"
+    ],
+    "modules": [
+        "shared-modules/qt4/qt4-4.8.7-minimal.json",
+        {
+            "name": "libedit",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.thrysoee.dk/editline/libedit-20191231-3.1.tar.gz",
+                    "sha256": "dbb82cb7e116a5f8025d35ef5b4f7d4a3cdd0a3909a146a39112095a2d229071"
+                }
+            ]
+        },
+        {
+            "name": "berkeleydb",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.oracle.com/berkeley-db/db-5.3.28.tar.gz",
+                    "sha256": "e0a992d740709892e81f9d93f06daf305cf73fb81b545afe72478043172c3628"
+                }
+            ],
+            "subdir": "dist",
+            "builddir": true
+        },
+        {
+            "name": "openldap",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.4.55.tgz",
+                    "sha256": "97357bd6ba8ff73d204c1f8208c5e6ab97d60c9e779f989057aa1229bbe98434"
+                }
+            ]
+        },
+        {
+            "name": "ike",
```

Comment:
Does the license allow redistribution? If not, it should use `extra-data`.

Reply:
Yes, the owner allows to do redistribution in its license. Is there any metadata for pointing it, apart of project_license?

Reply:
No, just making sure. :+1: